spec/std/isa/csr/misa.yaml

```
@@ -8,10 +8,14 @@ kind: csr
 name: misa
 long_name: Machine ISA Control
 address: 0x301
+# writable when misa CSR is implemented; read-only-0 otherwise
 writable: true
 priv_mode: M
 length: MXLEN
-description: Reports the XLEN and "major" extensions supported by the ISA.
+description: |
+  Reports the XLEN and "major" extensions supported by the ISA.
+  [when,"MISA_CSR_IMPLEMENTED == false"] 
+  This CSR is read-only-0 (all bits hardwired to zero) and cannot be written.
```

We've update how this is handled:


```suggestion
  - id: csr-misa-purpose
    text: |
      Reports the XLEN and "major" extensions supported by the ISA.
    when:
      param:
        name: MISA_CSR_IMPLEMENTED
        value: true
  - id: csr-misa-unimplemented
    text: |
      This CSR is read-only-0 (all bits hardwired to zero) and cannot be written.
    when:
      param:
        name: MISA_CSR_IMPLEMENTED
        value: false
```

Note that the schema will reject this until #891 ships